docs/canonicalk8s/snap/explanation/security/auth.md

```diff
@@ -0,0 +1,116 @@
+# Authentication and authorization in {{product}}
+
+{{product}} comes with a well-defined set of authentication and authorization
+mechanisms enabled by default. This guide provides an overview of these built-in
+security features, helping users understand what to expect out of the box and
+where to find further information for custom configuration.
+
+## Overview
+
+{{product}} leverages upstream Kubernetes security primitives for both
+authentication (identity verification) and authorization (determining
+permissions). By default, {{product}} includes:
+
+- TLS certificates for secure communication between components
+- Service accounts and service account tokens for workloads
+- Optional integration paths for external identity providers
+- Role-Based Access Control (RBAC)
+- Admission controllers
+
+## Authentication
+
+Authentication in {{product}} ensures that users and components are who they
+claim to be. The following methods are included by default:
+
+#### Client certificates
+
+- The Kubernetes API server is configured to trust specific client certificates.
+- Certificates are issued to admin users during cluster creation.
+- These can be seen by running `k8s config` on a control plane node.
+
+#### Service accounts
+
+- Every pod in Kubernetes is automatically assigned a service account, unless
+  specified otherwise.
+- Service account tokens are mounted into pods, enabling them to authenticate
+  with the API server securely, unless specified otherwise.
+- A policy engine (such as Kyverno) may restrict automounting the default
+  service account token into Pods if used.
+- These are managed in the namespace where the pod is deployed.
+
+The Kubernetes API server can be configured to accept OpenID Connect (OIDC)
+tokens for authentication from external identity providers.
+
+In {{product}}, anonymous API access is disabled by default.
+
+More details: [Kubernetes Authentication](https://kubernetes.io/docs/reference/access-authn-authz/authentication/)
```

Comment:
Nit: For more details, see [Kubernetes Authentication](https://kubernetes.io/docs/reference/access-authn-authz/authentication/) upstream docs.

Reply:
Done